Sources/StreamChatUI/CommonViews/Suggestions/ChatCommandSuggestionView/ChatCommandSuggestionCollectionViewCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A view cell that displays a command.
+public typealias ChatCommandSuggestionCollectionViewCell = _ChatCommandSuggestionCollectionViewCell<NoExtraData>
+
+/// A view cell that displays a command.
+open class _ChatCommandSuggestionCollectionViewCell<ExtraData: ExtraDataTypes>: _CollectionViewCell, ComponentsProvider {
```

Comment:
Does this need to be generic?...

Reply:
It is because it needs the `components` to get the `ChatCommangSuggestionView`